libs/blocks/chart/chart.js

```diff
@@ -487,22 +511,47 @@ export const getResponsiveSize = (authoredSize) => {
   return size;
 };
 
-/* c8 ignore next 19 */
+export const getLabelDegree = (chartStyles, isDesktop) => {
+  let diagonal = false;
+
+  if (isDesktop) {
+    diagonal = [...chartStyles]?.includes('desktop-diagonal-labels');
+  } else {
+    diagonal = [...chartStyles]?.includes('mobile-diagonal-labels');
+  }
+
+  return diagonal ? 60 : 0;
+};
+
+/* c8 ignore next 31 */
 const handleResize = (el, authoredSize, chartType, data, colors) => {
   const currentSize = getResponsiveSize(authoredSize);
   const previousSize = el?.getAttribute('data-responsive-size');
-  const previousIsLarge = previousSize === LARGE;
-  const currentIsLarge = currentSize === LARGE;
   const chartWrapper = el?.querySelector('.chart-wrapper');
   const chartInstance = window.echarts?.getInstanceByDom(chartWrapper);
+  const previousDevice = el?.getAttribute('data-device');
```

Comment:
I've not looked at where `handleResize` is used, but if `el` is the first parameter in the function and represents an element passed to the function, could you get rid of the optional chaining `?.` since we can assume a valid element is passed in? 

If you are worried about null, maybe better to make that explicit by adding
```
if (!el) return;
```
somewhere at the top of the function.